stylesheets/stylesheet.css

```diff
@@ -477,6 +477,33 @@ Custom Styles
   margin-bottom: 0;
 }
 
+.projects .label .count {
+  background: none white;
+  border: 1px solid white;
+  border-radius: 8px;
+  color: #0090ff;
+  display: inline-block;
+  height: 16px;
+  line-height: 16px;
+  margin: -2px -4px 0 4px;
```

Comment:
Oops, should have mentioned I was testing this in OS X.

:thumbsup: I'll chalk this one up to cross-browser hilarity and leave it for someone else to get sufficiently caremad about.